model_params:
  model_dir_path: /home/singh/ros_workspaces